cmd/podman/common.go

```diff
@@ -378,6 +378,7 @@ var createFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "shm-size",
 		Usage: "Size of `/dev/shm`. The format is `<number><unit>`. default is 64 MB",
```

Comment:
Can you describe unit suffixes here? We have a value ending in k here, which I assume is kb. Is m for MB valid?

Reply:
FWIW, I think this is fine as is here, but would love a full explanation in a man page if it's not there already.

Reply:
There is a full explanation in the man page.